public/pages/CreateTrigger/containers/CreateTrigger/utils/formikToTrigger.js

```diff
@@ -108,8 +108,7 @@ export function getDocumentLevelScriptSource(conditions) {
     }
     if (!_.isEmpty(query) && !_.isEmpty(query.queryName)) {
       const queryExpression = _.get(query, 'expression');
-      const operator = query.operator === '!=' ? '!' : '';
-      scriptSourceContents.push(`${operator}query[${queryExpression}]`);
+      scriptSourceContents.push(`query[${queryExpression}]`);
```

Comment:
Can you explain a little in the PR description, how removing this operator fixed the issue? Thanks

Reply:
Good idea! Added a `Bug fix explanation` to the PR details.